_data-prepper/pipelines/configuration/sources/s3.md

```diff
@@ -45,24 +45,54 @@ In order to use the `s3` source, configure your AWS Identity and Access Manageme
 
 If your S3 objects or Amazon SQS queues do not use [AWS Key Management Service (AWS KMS)](https://aws.amazon.com/kms/), remove the `kms:Decrypt` permission.
 
+## Cross-account S3 access
+
+### Bucket ownership<a name="s3_bucket_ownership"></a>
+
+When Data Prepper gets data from the S3 bucket, it verifies the ownership of the bucket using the 
+[bucket owner condition](https://docs.aws.amazon.com/AmazonS3/latest/userguide/bucket-owner-condition.html).
+By default, Data Prepper will expect that the S3 bucket is owned by the same account that owns the SQS queue.
+If not SQS queue is provided, it will use the role ARN in the `aws` configuration.
+
+If you are using cross-account S3 access, you need to configure how Data Prepper checks this condition.
+
+If all of your buckets are in one other account, you can specify the `default_bucket_owner` and set it to account Id of that account owning the other buckets.
+If your buckets are in multiple accounts, you should use the `bucket_owners` map.
+
+In this example, the SQS queue is owned by account `000000000000`. It will have data from two S3 buckets: `my-bucket-01` and `my-bucket-02`.
+These buckets are owned by different accounts - `my-bucket-01` is owned by `123456789012` and `my-bucket-02` is owned by `999999999999`.
+You can configure Data Prepper using the following.
+
+```
+s3:
+  sqs:
+      queue_url: "https://sqs.us-east-1.amazonaws.com/000000000000/MyQueue"
+  bucket_owners:
+    my-bucket-01: 123456789012
+    my-bucket-02: 999999999999
+```
+
+You can use both the `bucket_owners` and `default_bucket_owner` together.
 
 ## Configuration
 
 You can use the following options to configure the `s3` source.
 
-Option | Required | Type | Description
-:--- | :--- | :--- | :---
-notification_type | Yes | String | Must be `sqs`.
-compression | No | String | The compression algorithm to apply: `none`, `gzip`, or `automatic`. Default value is `none`.
-codec | Yes | Codec | The [codec](#codec) to apply. 
-sqs | Yes | sqs | The SQS configuration. See [sqs](#sqs) for details.
-aws | Yes | aws | The AWS configuration. See [aws](#aws) for details.
-on_error | No | String | Determines how to handle errors in Amazon SQS. Can be either `retain_messages` or `delete_messages`. If `retain_messages`, then Data Prepper will leave the message in the Amazon SQS queue and try again. This is recommended for dead-letter queues. If `delete_messages`, then Data Prepper will delete failed messages. Default value is `retain_messages`.
+Option | Required | Type     | Description
+:--- | :--- |:---------| :---
+notification_type | Yes | String   | Must be `sqs`.
+compression | No | String   | The compression algorithm to apply: `none`, `gzip`, or `automatic`. Default value is `none`.
+codec | Yes | Codec    | The [codec](#codec) to apply. 
+sqs | Yes | sqs      | The SQS configuration. See [sqs](#sqs) for details.
+aws | Yes | aws      | The AWS configuration. See [aws](#aws) for details.
```

Comment:
```suggestion
aws | Yes | AWS | The AWS configuration. See [aws](#aws) for details.
```

Reply:
I think the lowercase `aws` makes sense here because it is related to the `aws` configuration which is linked below.

Also, this is not related to the current PR.